packages/paste-website/src/components/DoDont.tsx

```diff
@@ -1,81 +1,67 @@
 import * as React from 'react';
 import styled from '@emotion/styled';
 import {themeGet} from 'styled-system';
+import {Absolute} from '@twilio-paste/absolute';
 import {Box} from '@twilio-paste/box';
 import {Text} from '@twilio-paste/text';
 
+interface DoDontProps {
+  children: React.ReactNode;
+}
+
 const StyledWrapper = styled(Box)`
   display: grid;
   grid-template-columns: repeat(2, 1fr);
   grid-gap: ${themeGet('space.space70')};
 `;
 
-interface DoDontProps {
-  children: React.ReactNode;
-}
-
 const DoDont: React.FC<DoDontProps> = props => {
   return (
-    <StyledWrapper marginTop="space90" marginBottom="space90">
+    <StyledWrapper marginTop="space90" marginBottom="space130">
       {props.children}
     </StyledWrapper>
   );
 };
 
-interface ExampleProps {
-  children: React.ReactNode;
-  do: boolean;
-  image?: string;
-}
-
-const StyledExampleWraper: React.FC<ExampleProps> = styled(Box)`
-  border-top: ${(props: ExampleProps) =>
-    props.do
-      ? `${themeGet('borderWidths.borderWidth20')(props)} solid #23bf6e`
-      : `${themeGet('borderWidths.borderWidth20')(props)} solid #ce241a`};
+const DoDontImage = styled(Box)`
+  border: ${themeGet('borderWidths.borderWidth10')} solid #ccd2dc;
+  border-bottom: 0;
 `;
 
-const StyledExampleImg: React.FC<ExampleProps> = styled(Box)`
+const DoDontComponent = styled(Absolute)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
   border: ${themeGet('borderWidths.borderWidth10')} solid #ccd2dc;
-  border-top: 0;
+  border-bottom: 0;
+`;
 
-  img {
-    display: block;
-    width: 100%;
-  }
+const Do = styled(Box)`
```

Comment:
as discussed in DM, extension of `Box` here is a little strange as you're setting a top padding without actually using Box to set the padding top which it exposes. A little tidy up can be made in these components

Reply:
@SiTaggart @TheSisb and I refactored the do/don't components so the preview is composable. The title and body of the component are build via props since those are used in a very specific way for this component.

I've also updated the `anchor` and `button` docs.